app/uk/gov/hmrc/economiccrimelevyregistration/models/audit/OtherEntityTypeSelectedEvent.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.models.audit
+
+import play.api.libs.json.{JsValue, Json, OFormat}
+import uk.gov.hmrc.economiccrimelevyregistration.models.OtherEntityType
+
+case class OtherEntityTypeSelectedEvent(internalId: String, entityType: OtherEntityType) extends AuditEvent {
```

Comment:
Get rid of this event/class for now

Reply:
We will need it and it does not hurt to have it there when the audit event is agreed.

Reply:
As above - there will be a JIRA ticket to address this properly - please remove for now as we shouldn't be merging 'half complete/reminder' code into our production branch

Reply:
Removed.